README.md

```diff
@@ -747,8 +747,47 @@ peered with.
 
 `peer_region` defines the region of the remote VPC if it is not in the same region as Aiven VPC.
 
-Computed property `state` tells the current state of the VPC. This property cannot be
-set, only read.
+`state` is the state of the peering connection. This property is computed by Aiven 
+therefore cannot be set, only read. Where state can be one of: `APPROVED`, 
+`PENDING_PEER`, `ACTIVE`, `DELETED`, `DELETED_BY_PEER`, `REJECTED_BY_PEER` and 
+`INVALID_SPECIFICATION`. 
+
+Happy path of the `state` includes the following transitions of a VPC peering connection: 
+`APPROVED` -> `PENDING_PEER` -> `ACTIVE`.
+
+More details regarding each state:
+
+- `APPROVED` is the initial state after the user does a successful creation of a 
+peering connection resource via Terraform. 
+
+- `PENDING_PEER` appears when Aiven system has created one or more cloud resources from 
+the Aiven cloud account to the user's VPC/network that was specified by the peering 
+connection. This state means a VPC peering connection resource was successfully created 
+and in the pending state, and the user needs to perform a cloud provider-specific steps 
+in their cloud account to activate it. The state can change to `ACTIVE`, `DELETED`, 
+`DELETED_BY_PEER` and `REJECTED_BY_PEER`.
```

Comment:
This could say that "The connection enters the `PENDING_PEER` state from `APPROVED` once the Aiven platform has created a connection to the specified peer successfully in the cloud, but the connection is not active until the user completes the setup in their cloud account. The steps needed in the user cloud account depend on the used cloud provider"